_includes/selector.html

```diff
@@ -501,8 +506,8 @@
                     var pkgs = this.additional_pip_packages.map(libraryToPkg);
                 } else {
                     var pkgs = this.active_packages
-                        .filter(pkg => pkg !== "Choose Specific Packages" && pkg !== "cuSignal")
-                        .map(libraryToPkg);
+                    .filter(pkg => pkg !== "Choose Specific Packages")
+                    .map(libraryToPkg);
```

Comment:
The indent indicates line continuation. I'd keep it.
```suggestion
                        .filter(pkg => pkg !== "Choose Specific Packages")
                        .map(libraryToPkg);
```